tardis/montecarlo/base.py

```diff
@@ -302,11 +320,40 @@ def to_hdf(self, path_or_buf, path=''):
         """
         runner_path = os.path.join(path, 'runner')
         properties = ['output_nu', 'output_energy', 'nu_bar_estimator',
-                      'j_estimator', 'montecarlo_virtual_luminosity']
+                      'j_estimator', 'montecarlo_virtual_luminosity',
+                      'last_line_interaction_in_id',
+                      'last_line_interaction_out_id',
+                      'last_line_interaction_shell_id',
+                      'packet_luminosity', 'output_nu'
+                      ]
         to_hdf(path_or_buf, runner_path, {name: getattr(self, name) for name
                                           in properties})
         self.spectrum.to_hdf(path_or_buf, runner_path)
         self.spectrum_virtual.to_hdf(path_or_buf, runner_path,
                                      'spectrum_virtual')
         self.spectrum_reabsorbed.to_hdf(path_or_buf, runner_path,
                                         'spectrum_reabsorbed')
+
+    @classmethod
+    def from_config(cls, config):
+        if config.plasma.disable_electron_scattering:
+            logger.warn('Disabling electron scattering - this is not physical')
+            sigma_thomson = 1e-200 / (u.cm ** 2)
```

Comment:
The logic was actually copied as-is from current master: https://github.com/tardis-sn/tardis/blob/60d1d1dcdcb92e608f3b4b1e5fb2be429398314c/tardis/io/config_reader.py#L948-L948. 

Since I don't really know what sigma_thomson or cross sections are in this context, I'd like a confirmation on if we should change that. @wkerzendorf?